tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/decisiontree/RandomForest.scala

```diff
@@ -100,7 +101,13 @@ class RandomForestTrainer(validationPercentage: Double, numDecisionTrees: Int,
     * @return the induced random forest
     */
   override def apply(data: FeatureVectorSource): ProbabilisticClassifier = {
-    val subtreeFiles = Seq.fill(numDecisionTrees) {
+    import scala.concurrent.duration._
+    import scala.concurrent._
+    import scala.language.postfixOps
+    implicit val executionContext = ExecutionContext.fromExecutor(
```

Comment:
I don't know if you can. The initial size matches the number of processors you have. It's probably faster than specifying a thread pool - but if you're seeing better performance specifying something, document it here, and leave it as-is. :)